services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/service.py

```diff
@@ -108,8 +108,11 @@ def comparison_order(cls) -> Dict["ServiceState", int]:
 
 class RunningDynamicServiceDetails(ServiceDetails):
     boot_type: ServiceBootType = Field(
-        ...,
-        description="describes how the dynamic services was started (legacy=V0, modern=V2)",
+        ServiceBootType.V0,
```

Comment:
why doing this?
look in [https://github.com/ITISFoundation/osparc-simcore/blob/48cbc1c5192ee61b65119452a6e1360c030e5a85/services/director-v2/src/simcore_service_director_v2/modules/director_v0.py](https://github.com/ITISFoundation/osparc-simcore/blob/48cbc1c5192ee61b65119452a6e1360c030e5a85/services/director-v2/src/simcore_service_director_v2/modules/director_v0.py)
there you override the boot_type on line 163 for example. Not done on line 129... is that the issue?
If setting this default is the final fix, then please remove the others if they are unnecessary.